modules/ROOT/pages/troubleshooting-metrics-not-showing-for-applications.adoc

```diff
@@ -0,0 +1,16 @@
+= Metrics Not Showing for Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+If the metrics aren't showing for client applications on the *My Applications* details page, ensure that you have the *Monitoring Viewer* permissions. 
```

Comment:
One reason could be this one. they could have another issue

Reply:
Hi @ferferrarazzo Should we document other reasons?